File: app/src/main/java/com/example/dkdus/triptonature/model/Place.java

package com.example.dkdus.triptonature.model;

import androidx.room.Room;

import com.example.dkdus.triptonature.model.place_material.Response;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Place {

    @SerializedName("response")
    @Expose
    private Response response;

    public Response getResponse() {
        return response;
    }

    public void setResponse(Response response) {
        this.response = response;
    }

}

File: app/src/main/java/com/example/dkdus/triptonature/model/Place.kt
package com.example.dkdus.triptonature.model

import com.example.dkdus.triptonature.model.place_material.Response
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Place (
    @SerializedName("response")
    @Expose
    var response: Response
)